src/handwritten/_install_daqmx.py

```diff
@@ -0,0 +1,332 @@
+import click
+import contextlib
+import errno
+import json
+import logging
+import os
+import pathlib
+import subprocess
+import sys
+import tempfile
+import traceback
+from typing import List, Tuple, Optional
+import urllib.request
+
+if sys.platform.startswith("win"):
+   import winreg
+
+MODULE_DIR = pathlib.Path(__file__).parent
+METADATA_FILE = MODULE_DIR / "metadata.json"
+
+def _parse_version(version: str) -> Tuple[int, ...]:
+    """
+    Split the version string into a tuple of integers representing major, minor, and update parts.
+
+    Args:
+        version (str): The version string in the format "major.minor.update".
+
+    Returns:
+        tuple: A tuple of integers representing the major, minor, and update parts of the version.
+
+    >>> _parse_version("23.8.0")
+    (23, 8, 0)
+    >>> _parse_version("24.0.0")
+    (24, 0, 0)
+    >>> _parse_version("invalid_version")
+    Traceback (most recent call last):
+    ...
+    click.exceptions.ClickException: Invalid version format found
+    """
+    try:
+        return tuple(int(part) for part in version.split("."))
```

Comment:
[namedtuple](https://docs.python.org/3.10/library/collections.html?highlight=namedtuple#collections.namedtuple) would be good here.

Reply:
Honestly, I think `Tuple[int, ...]` is fine. I don't think we need to access/print field names, and it's not critical to limit to a max number of fields.